community-membership.md

```diff
@@ -296,5 +297,69 @@ with the project. When a contributor returns to being more active, they may be
 promoted back to their previous position at the discretion of the current
 maintainers following the process defined above.
 
+## Specification Sponsor
+
+Specification sponsors are trusted collaborators of the technical committee, and
+work to review, approve, and sponsor [opentelemetry-specification][] issues and
+PRs.
+
+Members are defined
+in [Community Members -> Specification and Protos](./community-members.md#specifications-and-proto)
+and are members of the `spec-sponsors` team (TODO: add link to github team when
+available). Specification sponsors SHOULD list
+their [areas of interest](./areas-of-interest.md).
+
+### Requirements
+
+Specification sponsors are trusted to drive specification work which can be
+complex and which highly impactful to downstream language SIGs and to the
+OpenTelemetry project as a whole. Approvals from specification sponsors count
+towards the approvals required to merge specification PRs. Therefore, there is a
+high bar to becoming a specification sponsor.
+
+- Contributor to OpenTelemetry project for at least 1 year
+- Has been the reviewer or author (through the sponsorship of another
+  specification sponsor) of at least 10 substantial PRs to specification or
+  related repositories
+- Demonstrated sound technical judgement, and a deep understanding of the
+  specification and the OpenTelemetry project
+- Nominated by technical committee member (
+  see [becoming a specification sponsor](#becoming-a-specification-sponsor))
+
+### Responsibilities and privileges
+
+- Review, respond to, and approve issues / PRs when tagged or when within areas
+  of interest (inactivity for more than 1 month when tagged may result in
```

Comment:
The statements within parenthesis sound scary and discouraging. How about add a new session about turning emeritus? 

Like:

```markdown
### Emeritus sponsors

Sponsors who do not have the time commitment anymore to fulfill their responsibilities are expected to request to be moved to emeritus sponsors. If needed, the TC might also vote to move a sponsor to emeritus.
```

This should cover both the inactivity and the lack of confidence scenarios.

Reply:
I don't disagree, but for context, this language was lifted from the approvers responsibilities of this document: https://github.com/open-telemetry/community/blame/main/community-membership.md#L165

Reply:
Updated to include. I think this is an improvement on the existing language. The existing sections can be updated to reflect this in a followup if needed.